### PR TITLE
Converting version to age

### LINKED ISSUE #12: Change "version" string to "age" string in subreddit /r/totallynotrobots
[metric_units](https://www.reddit.com/user/metric_units) is a sassy reddit bot that finds imperial units, and replies with a metric conversion. 

When it finds itself in [/r/totallynotrobots](https://www.reddit.com/r/totallynotrobots/comments/72u81b/i_very_much_enjoy_cracking_open_a_cold_one_with/dnlq0ag/), metric bot disguises itself as human because well... of course it's not a robot.

That being said, all humans have ages, whereas robots have version numbers. So this story is to change "v0.10.1" in metric bot's footer to  say "age0.10.1"

See [reply_maker.js](https://github.com/cannawen/metric_units_reddit_bot/blob/master/src/reply_maker.js) to get started!

### PR DESCRIPTION
**Pull request by nitinkgp23**
Closes #12

### CODE REVIEW

File: src/reply_maker.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function formatReply(comment, conversions) {
   if (comment['subreddit'].match(/^totallynotrobots$/i)) {
     species = "HUMAN";
     source =  "~~SOURCE~~";
-    version = "~~" + environment['version'].toUpperCase() + "~~";
+    version = "~~" + "age"+ environment['version'].slice(1,).toUpperCase() + "~~";
```
Comment (cannawen) [inline src/reply_maker.js]: Hey, this looks good but the `~~` will add strikethrough markdown which is no longer needed (it used to strike through the version number), can you remove those?

**Comment by cannawen** [comment]
Also, stupid technicality, but I don't think hacktoberfest counts PRs towards the t-shirt contest unless it's submitted between Oct 1-31. Given it's still September now, do you want to re-submit the PR in a few days?

**Comment by nitinkgp23** [comment]
Never mind :p . You can merge this now. I will create more PRs later :p

**Comment by cannawen** [comment]
My tests are not compiling when I try to run them (I should probably add some real CI to this repo, huh?), I think slice(1,) might not be valid syntax in Node? I think slice(1) would work

You can run the tests on your machine with `npm test`

```
/Users/canna/workspace/reddit-bot/src/reply_maker.js:15
    version = "age"+ environment['version'].slice(1,).toUpperCase();
                                                    ^
SyntaxError: Unexpected token )
    at Object.exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:513:28)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/canna/workspace/reddit-bot/test/reply_maker-test.js:4:17)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at /Users/canna/workspace/reddit-bot/node_modules/mocha/lib/mocha.js:230:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/canna/workspace/reddit-bot/node_modules/mocha/lib/mocha.js:227:14)
    at Mocha.run (/Users/canna/workspace/reddit-bot/node_modules/mocha/lib/mocha.js:495:10)
    at Object.<anonymous> (/Users/canna/workspace/reddit-bot/node_modules/mocha/bin/_mocha:469:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:456:3
npm ERR! Test failed.  See above for more details.
```

**Comment by cannawen** [comment]
I added CI, so the next time you push you should be able to see the Travis CI checks :)

**Pull request closed without merging**